src/react-components/src/components/02-molecules/SearchBar/SearchBar.stories.tsx

```diff
@@ -0,0 +1,16 @@
+import * as React from "react";
+
+import SearchBar from "./SearchBar";
+import { action } from "@storybook/addon-actions";
+import { withKnobs, text } from "@storybook/addon-knobs";
+
+
+export default {
+  title: "SearchBar",
+  component: SearchBar,
+};
+
+export const searchWithParams = () => <SearchBar searchBarId="searchBarId" searchButtonId="searchButtonId" dropdownId="dropdownId" dropdownOptions={["Keyword", "Other Keyword"]}
```

Comment:
These lines are long and I'd cut them back a bit.

Also, when you click on "seach" in storybook, the page changes. The `searchSubmitHandler` is being called, but since there's no preventing the default action, the page changes. You could change that in the `Button` component but not sure if that's a big deal, it's more for Storybook to stay on the same page and show the action when search is clicked.

Reply:
I'm gonna add a thing to the backlog to handle that in buttons - I've noticed it too, but I think it's in the real of storybook improvements that we want to come back to!